crates/driver/src/boundary/mempool.rs

```diff
@@ -28,20 +29,41 @@ use {
     web3::types::AccessList,
 };
 
+/// How to calculate the blockchain gas price when publishing a transaction to
+/// the mempool.
+#[derive(Debug, Clone)]
+pub enum GasPriceCalculation {
+    EthGasStation,
+    GasNow,
+    GnosisSafe,
+    Web3,
+    BlockNative { api_key: String },
+    Native,
+}
```

Comment:
This is quite complete! I do think that just hardcoding this to be `Native` should be good enough. We have multiple sources from pre-EIP-1559 gas estimates, but in the new driver, we probably don't need to support this.